modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -192,6 +192,81 @@ definition {
 			actionLabel = "Action Label");
 	}
 
+	@description = "LPS-178405 - Verify if actions works with oldValue function when trigger is On After Delete (Add an Object Entry)"
+	@priority = 5
+	test CanCreateActionAddAnObjectEntryUsingOldValueWithOnAfterDeleteTrigger {
+		task ("Given: Two custom objects with a text field") {
+			for (var objectNum : list "1,2") {
+				ObjectAdmin.addObjectViaAPI(
+					labelName = "Custom Object ${objectNum}",
+					objectName = "CustomObject${objectNum}",
+					pluralLabelName = "Custom Objects ${objectNum}");
+
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Text",
+					fieldLabelName = "Custom Text Field",
+					fieldName = "customObjectField",
+					fieldType = "String",
+					isRequired = "false",
+					objectName = "CustomObject${objectNum}");
+
+				ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject${objectNum}");
+			}
+		}
+
+		task ("And Given: The custom objects has a On Afer Delete Trigger action") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 1");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				active = "true",
+				fieldName = "Custom Text Field",
+				inputExpression = "customObjectField == 'Entry Test'",
```

Comment:
the inputExpression needs to have the oldValue function here. 

Should look something like _oldValue("customObjectField") == "Entry Test"_